source/modules/collection-api/authentication/overview.md.erb

```diff
@@ -1,18 +1,17 @@
 ## Authentication Activity
+```shell
+curl -XPOST https://collection.inspcdn.net/auth \
+    -d '{
+        "account_id": "232550",
+        "action":     "login_failure"
+    }'
+```
 
-The following functions provide Inspetor with information on when a user logs in our out of an account within your product:
+Account login/logout behavior is highly correlated with fraudulent activity. You can send information about successful or failed login attempts for an account (as well as logouts) via the `/auth` endpoint, which supports `POST` requests only.
 
-- <a href="#account-login">trackLogin</a>
-- <a href="#account-logout">trackLogout</a>
+The following fields are required:
 
-All functions for tracking authentication activity require an [InspetorAuth](#inspetorauth) object as an argument and return `true` upon successful execution. Otherwise, one of the following exceptions will be thrown:
-
-Exception | Description
---------- | -----------
-**[AuthException](#inspetorauthexception)**          | The Auth object provided is invalid
-**[generalException](#inspetorgeneralexception)**  | The timestamps passed as Auth object properties are invalid
-**[TrackerException](#trackerexception)**    | An internal error occurred (*This should never happen*)
-
-
-<p><%= partial "modules/collection-api/authentication/actions/login.md.erb" %></p>
-<p><%= partial "modules/collection-api/authentication/actions/logout.md.erb" %></p>
\ No newline at end of file
+Argument    | Type   | Description
+--------    | ----   | -----------
+account_id  | String | The account ID that was (or was attempted to be) logged into\
```

Comment:
Nit: to keep the style with the other ones the name of the column should be `Property` instead of `Argument`. Apart from that, in `event_datetimes` we put the fields in bold

Reply:
Good catch!